cetmix_tower_webhook/readme/CONFIGURE.md

```
@@ -0,0 +1,52 @@
+**Complete the following fields:**
+
+- Name. Authenticator name
+- Reference. Unique reference. Leave this field blank to auto generate it
+- Code. Code that is used to authenticate the request. You can use all Cetmix Tower - Python command variables except for the server​ plus the following webhook specific ones:
+- headers: dictionary that contain the request headers
+- raw_data: string with the request raw data
+- payload: dictionary that contains the json payload or the GET parameters of the request
+
+**The code returns the result​ variable in the following format:**
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)